client-src/elements/chromedash-metadata.ts

```diff
@@ -85,15 +88,17 @@ class ChromedashMetadata extends LitElement {
   }
 
   selectInVersionList(index) {
-    this.shadowRoot.getElementById('versionlist').selectedIndex = index; // log null
+    (
+      this.renderRoot.querySelector('#versionlist') as HTMLSelectElement
+    ).selectedIndex = index; // log null
   }
 
   _processResponse(response) {
     // TODO(ericbidelman): Share this data across instances.
     const windowsVersions = response[0];
     for (let i = 0, el; (el = windowsVersions.versions[i]); ++i) {
       // Include previous version if current is foobar'd.
-      this._channels[el.channel] =
+      this._channels[el.channel].version =
```

Comment:
Nothing assigns `this._channels[el.channel]` to be an object, so I'd expect this to throw when it's executed.